### PR TITLE
Remove plausible tracking from redirects

### PR DESCRIPTION
**Pull request by melissawm**
Closes napari/docs#190

cc @neuromusic

### CODE REVIEW

**Comment by melissawm** [comment]
Gentle ping @neuromusic @codemonkey800 😄

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "esbonio.sphinx.confDir": ""
+}
```
Comment (codemonkey800) [inline .vscode/settings.json]: hmm could we remove this? 🤔

**Review by codemonkey800 — APPROVED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (melissawm) [reply .vscode/settings.json]: oops sure, not sure how that went in

**Comment by neuromusic** [comment]
LGTM!

**Pull request merged**